lib/meadow/data/csv_export.ex

```diff
@@ -0,0 +1,98 @@
+defmodule Meadow.Data.CsvExport do
```

Comment:
```suggestion
defmodule Meadow.Data.CSVExport do
```

Any objections to capitalizing "CSV"? Thinking about `NimbleCSV` and how we went for `Meadow.IIIF` as well